src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -93,10 +93,11 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
     propertyNodes.foreach { propertyNode =>
       membersAndValues
         .filter { case (key, _) => propertyNode.name == key.code.slice(3, key.code.length - 2) }
-        .foreach { case (key, _) =>
-          println(s"Member edge ${key.code.slice(3, key.code.length - 2)} <-> ${propertyNode.value}")
-          builder.addEdge(propertyNode, key, EdgeTypes.IS_USED_AT)
-          builder.addEdge(key, propertyNode, EdgeTypes.ORIGINAL_PROPERTY)
+        .foreach { case (key, value) =>
+          println(s"Member edge str ${key.code.slice(3, key.code.length - 2)} <-> ${propertyNode.value}")
+          println(s"Member edge ${value.code} <-> ${propertyNode.value}")
```

Comment:
remove the println statement, if needed at a log debug

Reply:
Done